.github/workflows/clippy_bors.yml

```diff
@@ -0,0 +1,232 @@
+name: Clippy Test (bors)
+
+on:
+  push:
+    branches: [auto, try]
+    # Don't run tests, when only textfiles were modified
+    paths-ignore:
+    - 'COPYRIGHT'
+    - 'LICENSE-*'
+    - '**.md'
+    - '**.txt'
+
+env:
+  RUST_BACKTRACE: 1
+  CARGO_TARGET_DIR: '${{ github.workspace }}/target'
+  GHA_CI: 1
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: rust-lang/simpleinfra/github-actions/cancel-outdated-builds@master
+      with:
+        github_token: "${{ secrets.github_token }}"
+    - name: Checkout
+      uses: actions/checkout@v2.0.0
+      with:
+        ref: ${{ github.ref }}
+    - name: Check Changelog
+      run: |
+        MESSAGE=$(git log --format=%B -n 1)
```

Comment:
I think also this block of code would benefit from being in a separate script file

Reply:
I agree that extracting this in a separate file would be better. Sadly you can only access secrets in the `.yml` file and not in `.sh` files, that are called from the `.yml` file. At least, I don't know how to do this.

Reply:
Oh that's a bummer. I don't know too much about GHA, maybe it's possible to export the secret into an ENV var?

Reply:
I tried that, but then only `***` (or something similar) gets written in the env var, for security reasons.

We could do something like I did in 531ee0a, but in this case it would be pretty much the whole script and in the extracted `.sh` script only the `if ...` would be extracted.